charts/stable/sonarr/values.yaml

```diff
@@ -40,7 +40,6 @@ workload:
               path: /ping
           env:
             SONARR__PORT: "{{ .Values.service.main.ports.main.port }}"
-            SONARR__AUTHENTICATION_METHOD: ""
```

Comment:
revert this :D

Reply:
It is not in the other arrs, i thought it was only needed in sonarr v3.

Reply:
It should be as long as it’s on questions too.